packages/cms/pages/auth/register/register.tsx

```diff
@@ -31,36 +49,58 @@ export const Register: React.FunctionComponent = () => {
 
       <EuiSpacer size="xl" />
 
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow label="First name">
-            <EuiFieldText autoFocus />
-          </EuiFormRow>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow label="Last name">
-            <EuiFieldText />
-          </EuiFormRow>
-        </EuiFlexItem>
-      </EuiFlexGroup>
+      <form method="post" onSubmit={submitRegistration}>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFormRow label="First name">
+              <EuiFieldText
+                autoFocus
+                onChange={e => {
```

Comment:
`event` or `changeEvent`. Sometimes the things that mean something to us are absolutely confusing to other people reading the code.

Reply:
Thanks